src/Korzh.DbUtils.Postgre/PostgreBridge.cs

```diff
@@ -0,0 +1,198 @@
+﻿using Microsoft.Extensions.Logging;
+using Npgsql;
```

Comment:
It would be great if you follow such order of namespaces. 
System namespaces....

Microsoft namespaces....

Other namespaces....

Korzh namespaces...